src/thirdparty/easylogging++.cc

```diff
@@ -29,90 +29,92 @@ namespace consts {
 
 // Level log values - These are values that are replaced in place of %level format specifier
 // Extra spaces after format specifiers are only for readability purposes in log files
-static const base::type::char_t *kInfoLevelLogValue    = ELPP_LITERAL("INFO");
```

Comment:
You can move these files, but the formatting should remain the same.

Reply:
I have checked it again this is the original version from commit  : 02f4b1cd9cf2be93408ed84c4488326b9579e52c 

It is reformatted in 2a767ebe5552031ed931a232b6300415de1ef6b0

I believe we should keep the original version